docs/internals/contributing/triaging-tickets.txt

```diff
@@ -114,39 +114,40 @@ Beyond that there are several considerations:
 
 * **Accepted + No Flags**
 
-  The ticket is valid, but no one has submitted a patch for it yet. Often this
-  means you could safely start writing a fix for it. This is generally more
-  true for the case of accepted bugs than accepted features. A ticket for a bug
-  that has been accepted means that the issue has been verified by at least one
-  triager as a legitimate bug - and should probably be fixed if possible. An
-  accepted new feature may only mean that one triager thought the feature would
-  be good to have, but this alone does not represent a consensus view or imply
-  with any certainty that a patch will be accepted for that feature. Seek more
-  feedback before writing an extensive contribution if you are in doubt.
+  The ticket is valid, but no one has submitted a pull request for it yet.
+  Often this means you could safely start writing a fix for it. This is
+  generally more true for the case of accepted bugs than accepted features. A
+  ticket for a bug that has been accepted means that the issue has been
+  verified by at least one triager as a legitimate bug - and should probably be
+  fixed if possible. An accepted new feature may only mean that one triager
+  thought the feature would be good to have, but this alone does not represent
+  a consensus view or imply with any certainty that a pull request will be
+  accepted for that feature. Seek more feedback before writing an extensive
+  contribution if you are in doubt.
 
-* **Accepted + Has Patch**
+* **Accepted + Has Pull Request**
 
   The ticket is waiting for people to review the supplied solution. This means
-  downloading the patch and trying it out, verifying that it contains tests
-  and docs, running the test suite with the included patch, and leaving
-  feedback on the ticket.
+  checking out the pull request and trying it out, verifying that it contains
+  tests and docs, checking that all the automated tests on the pull request are
+  passing, and leaving feedback on the ticket.
```

Comment:
```suggestion
  passing, and leaving feedback.
```
As feedback might be better suited on the PR maybe

Reply:
I feel like I'm repeating myself, but yes, great point 👍🏻